skbio/embedding/_protein.py

```diff
@@ -96,5 +96,5 @@ def __repr__(self):
 
 
 example_protein_embedding = ProteinEmbedding(
-    np.load(np.random.randn(62, 1024)),
+    np.random.randn(62, 1024),
```

Comment:
`randn` might be replaced by [`standard_normal`](https://numpy.org/doc/stable/reference/random/generated/numpy.random.Generator.standard_normal.html) in future NumPy versions.